src/views/index/index.jsx

```diff
@@ -81,19 +81,19 @@ const Index = () => {
                 {/* end content-graphic */}
                 <div className="content-news">
                     <div className="content-news-header">
-                        News
+                       News
                     </div>
                     <div className="content-news-body">
                         <a
-                            href="https://medium.com/scratchfoundation-blog/10-things-to-try-right-now-with-your-child-on-scratch-and-scratchjr-defd7f8ad0d1"
+                            href="https://www.amazon.com/Official-ScratchJr-Book-Help-Learn/dp/1593276710/"
                         >
-                        10 things to try right now with your child on Scratch and ScratchJr
+                      The Official ScratchJr Book is the perfect companion to the app and makes coding easy and fun for all
```

Comment:
Close the `</a>` after 'Book'. That is replace the line below with:
`</a> is the perfect companion to the app and makes coding easy and fun for all.`, and remove the duplicate part of the text above.